.github/workflows/after-merge.yml

```diff
@@ -0,0 +1,157 @@
+# Run after merge to trunk
+on:
+  push:
+    branches: [master] # $default-branch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['14.x', '16.x']
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      # 'yarn install' must be done at the top level, to build all the
+      # cross-package symlinks
+      - name: yarn install
+        run: yarn install
+      - name: check if git is dirty
+        run: git diff --stat --exit-code
+      - name: check dependencies
+        run: yarn check-dependencies
```

Comment:
Drive by, but why use `check-dependencies` instead of `--frozen-lockfile`

Reply:
In the version of Yarn we are using, `--frozen-lockfile` exited successfully, even if the lockfile would have been changed.